examples/matlab/KG_product.m

```diff
@@ -1,23 +1,23 @@
-% Tests flux operator 'KG' aka tensor times the gradient
-clc
-close all
-
-addpath('../../src/mole_MATLAB')
-
-% Parameters
-k = 2;
-m = 10;
-n = 7;
-
-%K = ones(2, 2);
-
-Kxx = rand(m*n+n, 1);
-Kyy = rand(m*n+m, 1);
-Kxy = rand(m*n+m, 1);
-Kyx = rand(m*n+n, 1);
-
-K = {Kxx Kyy Kxy Kyx};
-
-G = grad2D(k, m, 1, n, 1);
-
-KG = tensorGrad2D(K, G);
+% Tests flux operator 'KG' aka tensor times the gradient
+clc
+close all
+
+addpath('../../src/mole_MATLAB')
```

Comment:
This folder no longer exists. The mole source directory is hard coded into all of the MATLAB examples. If you change the name, all the MATLAB files need the reference changed.